src/components/CustomerQuota/TransactionsGroup.tsx

```diff
@@ -50,7 +50,7 @@ export interface Transaction {
 }
 
 export interface TransactionsGroup {
-  header: string;
+  header: string | undefined;
```

Comment:
Could you explain when the header will be undefined?

Reply:
based on the eida's UI, she wants the list to show `transactionsByCategoryList` without showing the category header (in this case is `Lost Token`). To handle this, I added `header={undefined}` to remove the header in `AddonsItems`. What do you thinkk?